caput/memh5.py

```diff
@@ -411,7 +425,7 @@ def from_hdf5(
         selections=None,
         convert_dataset_strings=False,
         convert_attribute_strings=True,
-        **kwargs
+        **_,
```

Comment:
Rick is not passing on the `kwargs` here, I am not sure why, yet. Open question if this that we want to do?

Reply:
The more I look at this, the more I think that we do want to pass on the `kwargs`, and removing them was possibly a linting error.